cases/templates/flying_wings.py

```diff
@@ -1104,6 +1104,272 @@ def update_mass_stiff(self):
 
+class Pazi(FlyingWing):
+    '''
+    Build a Pazi wing.
+    This class is nothing but a FlyingWing with pre-defined geometry properties
+    and mass/stiffness data ("update_mass_stiffness" method)
+    '''
+
+    def __init__(self,
+                 M, N,  # chord/span-wise discretisations
+                 Mstar_fact,
+                 u_inf,  # flight cond
+                 alpha,
+                 rho=1.225,
+                 b_ref=2. * 0.55,  # geometry
+                 main_chord= 0.1,
+                 aspect_ratio=(2. * 0.55) / 0.1,
+                 roll=0.,
+                 yaw=0.,
+                 beta=0.,
+                 sweep=0.,
+                 n_surfaces=1,
+                 physical_time=2,
+                 route='.',
+                 case_name='pazi',
+                 RollNodes=False):
+        super().__init__(M=M, N=N,
+                         Mstar_fact=Mstar_fact,
+                         u_inf=u_inf,
+                         alpha=alpha,
+                         rho=rho,
+                         b_ref=b_ref,
+                         main_chord=main_chord,
+                         aspect_ratio=aspect_ratio,
+                         roll=roll,
+                         beta=beta,
+                         yaw=yaw,
+                         sweep=sweep,
+                         physical_time=physical_time,
+                         n_surfaces=n_surfaces,
+                         route=route,
+                         case_name=case_name,
+                         RollNodes=RollNodes)
+
+        # aeroelasticity parameters
+#         self.main_ea = 0.3
+        self.main_ea = 0.4475
+        self.main_cg = 0.4510
+        self.sigma = 1
+
+        # other
+        self.c_ref = main_chord
+
+    def update_mass_stiff(self):
+        '''
+        This method can be substituted to produce different wing configs.
+
+        For this model, remind that the delta_frame_of_reference is chosen such
+        that the B FoR axis are:
+        - xb: along the wing span
+        - yb: pointing towards the leading edge (i.e. roughly opposite than xa)
+        - zb: upward as za
+        '''
+        # uniform mass/stiffness
+
+        # Pulling:
+        ea = 7.12E+07
+
+        # In-plane bending:
+        ga_inp = 3.31E+06
+        ei_inp = 3.11E+03
+
+        # Out-of-plane bending:
+        # Original:
+        # ga_oup = -4.16E+03
+        ei_oup = 4.67E+00
+
+        # Adjusted:
+        # Adjusted to become non-negative
+        ga_oup = 1E+06
+
+        # Torsion:
+        gj = 7.20E+00
+
+        base_stiffness = np.diag([ea, ga_oup, ga_inp, gj, ei_oup, ei_inp])
+
+        self.stiffness = np.zeros((1, 6, 6))
+        self.stiffness[0] = base_stiffness
+
+        m_unit = 0.589 # kg/m
+
+        # Lumped mass to confirm wing properties:
+        # Bending:
+        # 5 N load at the tip. Expected deflection of about 62.6 mm at the tip (no gravity)
```

Comment:
This looks like a good future verification test. Would you mind adding a bit more detail on how to achieve that result? What discretisation and angle of attack did you use to achieve this?

Reply:
@ngoiz, I've added some details on the tests. They are verification tests against a full 3D FEA analysis in Abaqus